content/cloud-docs/api-docs/providers.mdx

```diff
@@ -336,6 +726,559 @@ curl \
+
+Properties without a default value are required.
+
+| Key path                   | Type   | Default  | Description                              |
+| -------------------------  | ------ | -------  | -------------------------------------    |
+| `data.type`                | string |          | Must be `"registry-provider-platforms"`. |
+| `data.attributes.os`       | string |          | A valid semver version string.           |
+| `data.attributes.arch`     | string |          | A valid os string.                       |
+| `data.attributes.shasum`   | string |          | A valid shasum string.                   |
+| `data.attributes.filename` | string |          | A valid filename string.                 |
+
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type": "registry-provider-version-platforms",
+    "attributes": {
+      "os": "linux",
+      "arch": "amd64",
+      "shasum": "8f69533bc8afc227b40d15116358f91505bb638ce5919712fbb38a2dec1bba38",
+      "filename": "terraform-provider-aws_3.1.1_linux_amd64.zip"
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions/3.1.1/platforms
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "provpltfrm-BLJWvWyJ2QMs525k",
+    "type": "registry-provider-platforms",
+    "attributes": {
+      "os": "linux",
+      "arch": "amd64",
+      "filename": "terraform-provider-aws_3.1.1_linux_amd64.zip",
+      "shasum": "8f69533bc8afc227b40d15116358f91505bb638ce5919712fbb38a2dec1bba38",
+      "permissions": {
+        "can-delete": true,
+        "can-upload-asset": true
+      },
+      "provider-binary-uploaded": false
+    },
+    "relationships": {
+      "registry-provider-version": {
+        "data": {
+          "id": "provver-y5KZUsSBRLV9zCtL",
+          "type": "registry-provider-versions"
+        }
+      }
+    },
+    "links": {
+      "provider-binary-upload": "https://archivist.terraform.io/v1/object/dmF1b..."
+    }
+  }
+}
+
+```
+
+## Get all platforms for a single version (private provider)
+
+`GET /organizations/:organization_name/registry-providers/:registry_name/:namespace/:name/versions/:version/platforms`
+
+### Parameters
+
+| Parameter            | Description                                                                                              |
+| -------------------- | -------------------------------------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization the provider belongs to.                                                    |
+| `:namespace`         | The namespace of the provider. Must be the same as the `organization_name` for the provider              |
```

Comment:
```suggestion
| `:namespace`         | The namespace of the provider. Must be the same as the `organization_name` for the provider.              |
```

Reply:
@madaleines we're inconsistent about whether we include the colon when we're talking about the parameter or not. I've been removing it when we've been talking about it in the description, but please let me know if it should stay. Here for example, it's not included.